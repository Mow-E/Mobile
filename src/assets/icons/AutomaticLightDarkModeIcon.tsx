import * as React from 'react';
import Svg, {SvgProps, Path, Circle, Mask, Line} from 'react-native-svg';
import IconProps from './IconProps';
import colors from '../../styles/colors';

function SvgComponent(props: SvgProps): JSX.Element {
  return (
    <Svg width={24} height={24} viewBox="0 0 40 39" fill="none" {...props}>
      <Circle
        cx={20.5}
        cy={19.5}
        r={8}
        transform="rotate(-90 20.5 19.5)"
        fill={colors.gray['50']}
        stroke={props.stroke}
      />
      <Mask id="path-2-inside-1_67_4" fill={colors.gray['50']}>
        <Path d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z" />
      </Mask>
      <Path
        d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
        fill={colors.gray['950']}
        stroke={props.stroke}
        strokeWidth={2}
        mask="url(#path-2-inside-1_67_4)"
      />
      <Line x1={20.5} y1={10} x2={20.5} stroke={props.stroke} />
      <Line x1={20.5} y1={39} x2={20.5} y2={29} stroke={props.stroke} />
      <Line x1={30} y1={20.5} x2={40} y2={20.5} stroke={props.stroke} />
      <Line
        x1={27.3796}
        y1={27.6746}
        x2={33.8875}
        y2={35.2672}
        stroke={props.stroke}
      />
      <Line
        x1={35.3536}
        y1={5.35355}
        x2={28.3536}
        y2={12.3536}
        stroke={props.stroke}
      />
      <Path d="M12.5 27.5L5.5 34.5" stroke={props.stroke} />
      <Line
        x1={6.37963}
        y1={4.6746}
        x2={12.8875}
        y2={12.2672}
        stroke={props.stroke}
      />
      <Line y1={20.5} x2={10} y2={20.5} stroke={props.stroke} />
    </Svg>
  );
}

/**
 * An icon for the automatic app mode.
 */
function AutomaticLightDarkModeIcon({size = 24}: IconProps): JSX.Element {
  return (
    <SvgComponent height={size} width={size} stroke={colors.gray['950']} />
  );
}

export default AutomaticLightDarkModeIcon;
