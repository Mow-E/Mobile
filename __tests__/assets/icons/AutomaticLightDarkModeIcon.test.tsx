import 'react-native';
import React from 'react';
import AutomaticLightDarkModeIcon from '../../../src/assets/icons/AutomaticLightDarkModeIcon';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders default correctly', () => {
  const tree = renderer.create(<AutomaticLightDarkModeIcon />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders colored correctly', () => {
  const tree = renderer.create(<AutomaticLightDarkModeIcon colored />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders uncolored correctly', () => {
  const tree = renderer
    .create(<AutomaticLightDarkModeIcon colored={false} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders in dark mode correctly', () => {
  const tree = renderer
    .create(<AutomaticLightDarkModeIcon darkModeInverted />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders in different sizes', () => {
  const treeSmall = renderer
    .create(<AutomaticLightDarkModeIcon size={24} />)
    .toJSON();
  const treeLarge = renderer
    .create(<AutomaticLightDarkModeIcon size={48} />)
    .toJSON();

  expect(treeSmall).not.toEqual(treeLarge);
});
