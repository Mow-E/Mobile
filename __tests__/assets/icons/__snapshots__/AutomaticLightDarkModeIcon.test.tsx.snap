// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders colored correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={39}
  vbWidth={40}
  width={24}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGCircle
      cx="20.5"
      cy="19.5"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      matrix={
        [
          6.123233995736766e-17,
          -1,
          1,
          6.123233995736766e-17,
          1,
          40,
        ]
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="8"
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGMask
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      height="100%"
      maskContentUnits={1}
      maskUnits={0}
      name="path-2-inside-1_67_4"
      propList={
        [
          "fill",
        ]
      }
      width="100%"
      x="0%"
      y="0%"
    >
      <RNSVGPath
        d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
    </RNSVGMask>
    <RNSVGPath
      d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      mask="path-2-inside-1_67_4"
      propList={
        [
          "fill",
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="2"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="10"
      y2="0"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="39"
      y2="29"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="30"
      x2="40"
      y1="20.5"
      y2="20.5"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="27.3796"
      x2="33.8875"
      y1="27.6746"
      y2="35.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="35.3536"
      x2="28.3536"
      y1="5.35355"
      y2="12.3536"
    />
    <RNSVGPath
      d="M12.5 27.5L5.5 34.5"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="6.37963"
      x2="12.8875"
      y1="4.6746"
      y2="12.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="0"
      x2="10"
      y1="20.5"
      y2="20.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders default correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={39}
  vbWidth={40}
  width={24}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGCircle
      cx="20.5"
      cy="19.5"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      matrix={
        [
          6.123233995736766e-17,
          -1,
          1,
          6.123233995736766e-17,
          1,
          40,
        ]
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="8"
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGMask
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      height="100%"
      maskContentUnits={1}
      maskUnits={0}
      name="path-2-inside-1_67_4"
      propList={
        [
          "fill",
        ]
      }
      width="100%"
      x="0%"
      y="0%"
    >
      <RNSVGPath
        d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
    </RNSVGMask>
    <RNSVGPath
      d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      mask="path-2-inside-1_67_4"
      propList={
        [
          "fill",
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="2"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="10"
      y2="0"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="39"
      y2="29"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="30"
      x2="40"
      y1="20.5"
      y2="20.5"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="27.3796"
      x2="33.8875"
      y1="27.6746"
      y2="35.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="35.3536"
      x2="28.3536"
      y1="5.35355"
      y2="12.3536"
    />
    <RNSVGPath
      d="M12.5 27.5L5.5 34.5"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="6.37963"
      x2="12.8875"
      y1="4.6746"
      y2="12.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="0"
      x2="10"
      y1="20.5"
      y2="20.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders in dark mode correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={39}
  vbWidth={40}
  width={24}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGCircle
      cx="20.5"
      cy="19.5"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      matrix={
        [
          6.123233995736766e-17,
          -1,
          1,
          6.123233995736766e-17,
          1,
          40,
        ]
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="8"
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGMask
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      height="100%"
      maskContentUnits={1}
      maskUnits={0}
      name="path-2-inside-1_67_4"
      propList={
        [
          "fill",
        ]
      }
      width="100%"
      x="0%"
      y="0%"
    >
      <RNSVGPath
        d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
    </RNSVGMask>
    <RNSVGPath
      d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      mask="path-2-inside-1_67_4"
      propList={
        [
          "fill",
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="2"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="10"
      y2="0"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="39"
      y2="29"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="30"
      x2="40"
      y1="20.5"
      y2="20.5"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="27.3796"
      x2="33.8875"
      y1="27.6746"
      y2="35.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="35.3536"
      x2="28.3536"
      y1="5.35355"
      y2="12.3536"
    />
    <RNSVGPath
      d="M12.5 27.5L5.5 34.5"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="6.37963"
      x2="12.8875"
      y1="4.6746"
      y2="12.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="0"
      x2="10"
      y1="20.5"
      y2="20.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders uncolored correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={39}
  vbWidth={40}
  width={24}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGCircle
      cx="20.5"
      cy="19.5"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      matrix={
        [
          6.123233995736766e-17,
          -1,
          1,
          6.123233995736766e-17,
          1,
          40,
        ]
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="8"
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGMask
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      height="100%"
      maskContentUnits={1}
      maskUnits={0}
      name="path-2-inside-1_67_4"
      propList={
        [
          "fill",
        ]
      }
      width="100%"
      x="0%"
      y="0%"
    >
      <RNSVGPath
        d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
    </RNSVGMask>
    <RNSVGPath
      d="M20.5 28C22.7543 28 24.9163 27.1045 26.5104 25.5104C28.1045 23.9163 29 21.7543 29 19.5C29 17.2457 28.1045 15.0837 26.5104 13.4896C24.9163 11.8955 22.7543 11 20.5 11L20.5 19.5V28Z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      mask="path-2-inside-1_67_4"
      propList={
        [
          "fill",
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="2"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="10"
      y2="0"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="20.5"
      x2="20.5"
      y1="39"
      y2="29"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="30"
      x2="40"
      y1="20.5"
      y2="20.5"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="27.3796"
      x2="33.8875"
      y1="27.6746"
      y2="35.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="35.3536"
      x2="28.3536"
      y1="5.35355"
      y2="12.3536"
    />
    <RNSVGPath
      d="M12.5 27.5L5.5 34.5"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="6.37963"
      x2="12.8875"
      y1="4.6746"
      y2="12.2672"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="0"
      x2="10"
      y1="20.5"
      y2="20.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
